addons/event_hr_skills/models/event_registration.py

```diff
@@ -0,0 +1,42 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    def _create_resume_lines(self):
+        resume_line_vals_list = []
+        for registration in self:
+            if (
+                registration.state != 'done'
+                or not registration.event_id.tag_ids.category_id.resume_line_type_id
```

Comment:
Could probably groupby event_id, check it has any line type associated, then proceed with registrations of that event